orgs/branchprotection.yml

```diff
@@ -751,6 +751,24 @@ branch-protection:
             require_code_owner_reviews: true
             required_approving_review_count: 1
 
+        homebrew-tap:
+          allow_deletions: false
+          allow_disabled_policies: true
+          allow_force_pushes: false
+          enforce_admins: true
```

Comment:
Shouldn't this be `enforce_admins: false`?

Reply:
Thank you @stephanme. Good catch